hack/patch.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -xe
+
+source ./hack/common.sh
+
+kubectl=./cluster-up/kubectl.sh
+namespace=kubevirt
```

Comment:
With the above sourced script, you have `namespace` set to the right namespace automatically and you can remove this line.

Reply:
Done